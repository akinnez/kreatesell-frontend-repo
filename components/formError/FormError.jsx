import styles from "./FormError.module.scss";

export const FormError = ({ errors }) => {
	const error = Object.values(errors)?.map((data) => data);
	return (
		<div className={styles.error}>
			<p>{error?.[0]}</p>
		</div>
	);
};
