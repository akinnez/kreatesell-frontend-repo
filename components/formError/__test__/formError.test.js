import renderer from "react-test-renderer";
import { FormError } from "../FormError";

const mockErrors = {
  mockErrorOne: "isRequired",
  mockErrorTwo: "isRequired",
};
it("renders a snapshot of <FormError />", () => {
  const formErrorElement = renderer
    .create(<FormError errors={mockErrors} />)
    .toJSON();
  expect(formErrorElement).toMatchSnapshot();
});
