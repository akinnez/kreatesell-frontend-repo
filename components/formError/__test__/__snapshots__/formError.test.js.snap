// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <FormError /> 1`] = `
<div
  className="error"
>
  <p>
    isRequired
  </p>
</div>
`;
